tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -60,116 +52,134 @@ func (rb *RBACAdditionalTestSuite) SetupSuite() {
 	require.NoError(rb.T(), err)
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateAddStdUserAsProjectOwner() {
+func (rb *RBACAdditionalTestSuite) TestRbacP1ClusterOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
 
-	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	rb.T().Log("Set up a standard user for additional rbac test cases " + roleOwner)
```

Comment:
nit: Now that we have removed reference to cluster role, do we need 'roleOwner' in this log statement?

Reply:
Removed this log statement as it seems redundant